addons/web_editor/static/src/js/snippets.editor.js

```diff
@@ -453,7 +458,7 @@ data.Class = Widget.extend({
         var top = $tumb.outerHeight()/2;
         var $toInsert, dropped, $snippet;
 
-        $snippets.draggable({
+        $snippets.not('.o_snippet_install').draggable({
```

Comment:
Instead of excluding `.o_snippet_install` here, exclude them before this `make_snippet_draggable` function call. Also, I am not sure that this class is necessary, maybe the presence of `data-module-id` is a good enough selector.